### PR TITLE
Update DEFINITION.md

### PR DESCRIPTION
**Pull request by volaka**
Added Turkish (Türkçe) language

### CODE REVIEW

File: DEFINITION.md [hunk 1]
```diff
@@ -110,3 +110,11 @@ Le tecnologie **cloud native** permettono alle organizzazioni di costruire ed es
...
 
 La **Cloud Native Computing Foundation** cerca di favorire l'adozione di questo paradigma incoraggiando e sostenendo un ecosistema di progetti open source e indipendenti dai vendor. Noi democratizziamo le pratiche allo stato dell'arte per rendere queste innovazioni accessibili a tutti.
+
+## Türkçe
+
+Bulut yerlisi (Cloud Native) teknolojiler, kuruluşların, ölçeklenebilir uygulamalarını genel, özel ve hibrit bulutlar gibi modern ve dinamik ortamlarda oluşturup çalıştırmasını sağlar. Konteynerler, service mesh, mikroservisler, değişmez altyapı ve declarative API'ler bu yaklaşımı örneklendirir.
+
+Bu teknikler, esnek, yönetilebilir ve gözlemlenebilir olan birbirlerine az bağımlı sistemleri mümkün kılar. Sağlam bir otomasyon altyapısıyla birleştiğinde, mühendislerin minimum zahmetle sık ve öngörülebilir şekilde yüksek etkili değişiklikler yapmasına olanak tanır.
+
+**Cloud Native Computing Foundation**, açık kaynaklı, satıcı-bağımsız (vendor-neutral) projelerden oluşan bir ekosistemi teşvik ederek ve sürdürerek bu paradigmanın benimsenmesini sağlamayı amaçlamaktadır. Bu yenilikleri herkes için erişilebilir kılmak için, son teknoloji paternlerini demokratikleştiriyoruz.
```
Comment (OrlinVasilev) [inline DEFINITION.md:120]: last sentence must be : "Bu yenilikleri herkesin erişebilirsi için son teknoloji paternlerini demokratikleştiriyoruz."

**Review by OrlinVasilev — CHANGES_REQUESTED** [review]
last sentence must be : "Bu yenilikleri herkesin erişebilirsi için son teknoloji paternlerini demokratikleştiriyoruz."

**Comment by OrlinVasilev** [comment]
@volaka ^^ :)

**Comment by amye** [comment]
@volaka: can you accept changes? We're doing an end of year github cleanup, and I'd hate to close this if it's still relevant.

**Comment by volaka** [comment]
Hi, sorry I didn't see the request. I have changed it with better grammar. The requested change from @OrlinVasilev is not understandable. ("erişilebilirsi" is not a word :) )

**Comment by caniszczyk** [comment]
resubmit and deal with conflicts, sorry we let this one go stale :/

**Pull request closed without merging**